### ISSUE TITLE
[1.9.44] iPhone 6 Plus / 10.1.1, iPhone 6S / 10.0.2 - Entered profile information is not displayed under "Profile" section.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-ios-issues#973]
Frequency: Every time

Action:
1: Launch the app and login with valid credentials.
2: Tap on "Setting" icon.
3: Tap on any field under "Profile" section.e.g. "Display Name"
4: Enter valid display name and tap on tick to save it.
5: Check if entered display name is displayed under profile section.

Expected result: Entered profile information should be saved and displayed under "Profile" section.

Actual result: Entered profile information is not displayed under "Profile" section.

Note: issue is observed for all fields under "Profile" section.

**Comment by hiteshbheda** [comment]
@xiuie Similar to #968

**Comment by testershub** [comment]
@hiteshbheda This is not similar to #968. This is related to updating or changing the profile information.

**Comment by hiteshbheda** [comment]
@testershub Yes I agree with you, but it's common tendency that if profile information is not available at profile information page then how it is possible to fetch information during update / change specific content / fields. It really obvious. 

I know it's bug but we derive the value from information page as mentioned in #968
